friendica/content.md

```diff
@@ -0,0 +1,290 @@
+-	`FRIENDICA_LANG` The default language of the Friendica server.
+-	`FRIENDICA_PHP_PATH` The path of the PHP binary.
+-	`FRIENDICA_SITENAME` The Sitename of the Friendica server.
+-	`FRIENDICA_NO_VALIDATION` If set to `true`, the URL and E-Mail validation will be disabled.
+-	`FRIENDICA_DATA` If set to `true`, the fileystem will be used instead of the DB backend.
+-	`FRIENDICA_DATA_DIR` The data directory of the Friendica server (Default: /var/www/data).
+
+**Friendica Logging**
+
+-	`FRIENDICA_DEBUGGING` If set to `true`, the logging of Friendica is enabled.
+-	`FRIENDICA_LOGFILE` (optional) The path to the logfile (Default: /var/www/friendica.log).
+-	`FRIENDICA_LOGLEVEL` (optional) The loglevel to log (Default: notice).
+
+**Database** (**required at installation**)
+
+-	`MYSQL_USERNAME` Username for the database user using mysql.
+-	`MYSQL_USER` Username for the database user using mariadb.
+-	`MYSQL_PASSWORD` Password for the database user using mysql / mariadb.
+-	`MYSQL_DATABASE` Name of the database using mysql / mariadb.
+-	`MYSQL_HOST` Hostname of the database server using mysql / mariadb.
+-	`MYSQL_PORT` Port of the database server using mysql / mariadb (Default: `3306`)
+
+**Lock Driver (Redis)**
+
+-	`REDIS_HOST` The hostname of the redis instance (in case of locking).
+-	`REDIS_PORT` (optional) The port of the redis instance (in case of locking).
+-	`REDIS_PW` (optional) The password for the redis instance (in case of locking).
+-	`REDIS_DB` (optional) The database instance of the redis instance (in case of locking).
+
+## Administrator account
+
+Because Friendica links the administrator account to a specific mail address, you **have** to set a valid address for `MAILNAME`.
+
+## Mail settings
+
+The binary `ssmtp` is used for the `mail()` support of Friendica.
+
+You have to set the `--hostname/-h` parameter correctly to use the right domainname for the `mail()` command.
+
+You have to set a valid SMTP-MTA for the `SMTP` environment variable to enable mail support in Friendica. A valid SMTP-MTA would be, for example, `mx.example.org`.
+
+The following environment variables are possible for the SMTP examples.
+
+-	`SITENAME` The name of the Friendica node. (**required**)
+-	`SMTP` Address of the SMTP Mail-Gateway. (**required**)
+-	`SMTP_FROM` Sender user-part of the address. (Default: `no-reply` - e.g. no-reply@friendica.local)
+-	`SMTP_TLS` Use TLS for connecting the SMTP Mail-Gateway. (Default: empty)
+-	`SMTP_STARTTLS` Use STARTTLS for connecting the SMTP Mail-Gateway. (Default: empty)
+-	`SMTP_AUTH_USER` Username for the SMTP Mail-Gateway. (Default: empty)
+-	`SMTP_AUTH_PASS` Password for the SMTP Mail-Gateway. (Default: empty)
+-	`SMTP_AUTH_METHOD` Authentication method for the SMTP Mail-Gateway. (Default: empty/plain text)
+
+## Database settings
+
+You have to link a running database container, e. g. `--link my-mysql:mysql`, and then use `mysql` as the database host on setup.
```

Comment:
See also the note above regarding container linking -- ultimately your call although this reads as an even stronger "requirement" of linking that makes it sound like inter-container networking without `--link` is potentially unsupported, so I'd recommend at least rewording this somewhat to take that into account.